pages/bib/[id]/index.tsx

```diff
@@ -76,27 +89,62 @@ export default function BibPage({
     isArchiveCollection: bib.isArchiveCollection,
   })
 
-  const refreshItemTable = async (newQuery: BibQueryParams) => {
+  // Load all items via client-side fetch if page is first loaded with viewAllItems prop passed in
+  // Namely, when the page is accessed with the /all route
+  useEffect(() => {
+    if (viewAllItems) void refreshItemTable(query, true)
```

Comment:
What's the purpose of `void` here?

Reply:
That operator prevents an unhandled Promise warning.